### DISCUSSION TITLE
sdnext from 01. August - samplers missing ?

### DISCUSSION THREAD

**Discussion by cgidesign-de** [vladmandic/sdnext#1907] [Q&A] [answered]
Made a fresh git clone today:

This new install is missing several samplers that used to be in my previous install - see screenshot.
Anything I can do, to get the other samplers back (e,g, DPM 2a, or the Karras variants)?

![sdnext-samplers](https://github.com/vladmandic/automatic/assets/115958660/c527ebdc-7a16-4ef3-b942-3b089ff4fd14)

**Comment by vladmandic** [comment]
They are still there, but they are not separate schedulers (i.e. Karras is variant that can be applied to scheduler, not a different scheduler), see settings - samplers

**Comment by cgidesign-de** [comment]
Thanks for your help but I am still confused.
In the old install I had all the variants in the samplers dropdown of text 2 image and image 2 image.
Now there is only a limited subset as shown in the screenshot.

Do you mean I now need to change a setting in samplers and apply that change, to use a karras variant of a sampler instead of directly select it in the dropdown list?

If yes, would it be possible to get the old behaviour back? I was switching through a lot of samples varients frequently because they all tend to create a bit different output - some better for one type of image other better for anoter one.

For example the "a" samplers often have problems rendering 'monochrome' images while they are good color ones and so on.

![sdnext-samplers-2](https://github.com/vladmandic/automatic/assets/115958660/ed6d80ea-cae6-4209-bb7c-27ad1329e19b)

**Reply by vladmandic** [reply, marked as answer]
Your understanding is correct.
I plan to bring that part of settings to generate inferface, but I do not plan to bring old flat list back - that is just wrong as those are not actual samplers.

**Reply by cgidesign-de** [reply]
(I accidentally gave a thumbs up to my own post intead of yours - please ignore that)

**Comment by cgidesign-de** [comment]
Ok, got it. Thanks for explaining.
